### PR TITLE
Working code added with update in README.md

### PR DESCRIPTION
**Pull request by Shubby98**
There are some minor bugs in code due to which is not working in python 3.6 i fixed that.
and Some minor chnages in README.md file like which tensorflow version need to be installed and wikipedia needed to be installed too for working code

### CODE REVIEW

File: hub/examples/image_retraining/retrained_labels.txt [hunk 1]
```diff
@@ -7,4 +7,4 @@ uranus
 elliptical
 earth
 venus
-saturn
+saturn
```
Comment (ritwik12) [inline hub/examples/image_retraining/retrained_labels.txt]: @Shubby98 Just curious why is it showing a change? Is there whitespace?

**Comment by ritwik12** [comment]
@Shubby98 Please squash your commits and add a relative name to the commit and also PR name like `Code and README update for python3 and version change`

File: hub/examples/image_retraining/retrained_labels.txt (hunk 1 above)
Comment (Shubby98) [reply hub/examples/image_retraining/retrained_labels.txt]: @ritwik12  yeah there is whitespace after the saturn. actually i dont know how that file modified i never change it when i match it with original file i found order of planet on this file is changed so i just copy paste that order with original file in order.

Comment (ritwik12) [reply hub/examples/image_retraining/retrained_labels.txt]: @shubby98 remove whitespace then

Comment (Shubby98) [reply hub/examples/image_retraining/retrained_labels.txt]: @ritwik12  i did

**Pull request closed without merging**